src/pages/components/Speakers.tsx

```diff
@@ -90,7 +91,9 @@ export const Speakers = () => {
                   </svg>
                 </div>
                 <h3 className="text-[13px] font-semibold leading-4 sm:text-base">
-                  {t(`${speaker}.title`)}
+                  <Link href={`/speakers/${speaker}`}>
+                    <a>{t(`${speaker}.title`)}</a>
+                  </Link>
```

Comment:
Upgrading in UX if user just `tab` though the page. Screen reader will read name of speaker, then name of the presentation